src/editor/Collaboration.js

```diff
@@ -0,0 +1,15 @@
+define(function (require, exports, module) {
+    "use strict";
+
+    var togetherjs = require("togetherjs");
+
+    function Collaboration() {
+        this.togetherjs = togetherjs;
+        var self = this;
+        window.setTimeout(function() {
```

Comment:
Added this over here to make sure that togetherjs loads after brackets has loaded completely. 
After getting loaded, togetherjs looks for all the codemirror instances to keep in sync.
Will probably move to something like
```js
window.onload
```
or
```js
document.onload
```
later.